eva/models/storage/batch.py

```diff
@@ -75,18 +58,14 @@ def columns(self):
     def column_as_numpy_array(self, column_name="data"):
         return np.array(self._frames[column_name])
 
-    def to_json(self):
-        obj = {
-            "frames": self._frames,
-            "batch_size": len(self),
-            "identifier_column": self._identifier_column,
-        }
-        return json.dumps(obj, cls=BatchEncoder)
+    def serialize(self):
+        obj = {"frames": self._frames, "batch_size": len(self)}
```

Comment:
What is the usage for `batch_size`? Can we `return PickleSerializer.serialize(self)`?

Reply:
For cases where we omit some rows, at least batch_size tells the row count